tests/common.go

```diff
@@ -4116,7 +4136,30 @@ func DeleteAzureBucket(bucketName string) {
 		fmt.Sprintf("Failed to delete container. Error: [%v]", err))
 }
 
-// DeleteBucket deletes bucket from the cloud
+// DeleteNfsSubPath delete subpath from nfs shared path.
+func DeleteNfsSubPath() {
+	// From ENV variables get NFS share details.
+	nfsServerAddress, nfsPath, nfsSubPath, _ := GetNfsCredsFromEnv()
+	// mount the nfs share to master node.
+	masterNode := node.GetMasterNodes()[0]
+	mountCmd := fmt.Sprintf("mkdir -p /tmp/nfsMount ; mount -t nfs %s:%s /tmp/nfsMount", nfsServerAddress, nfsPath)
```

Comment:
Let's break in down into 2 command, in the current case it will only have err for the second command.

Reply:
Done